yoti_python_sdk/multivalue.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from yoti_python_sdk.protobuf import protobuf
+
+
+def parse(multi_value_bytes):
+    from yoti_python_sdk import attribute_parser  # needed here (and not above) for Python 2.7 & 3.4 dependency handling
+
+    proto = protobuf.Protobuf()
+    multi_value_list = []
+    parsed_multi_value = proto.multi_value(multi_value_bytes)
+
+    for multi_value_item in parsed_multi_value.values:
+        value = attribute_parser.value_based_on_content_type(multi_value_item.data, multi_value_item.content_type)
+
+        if value.content_type == proto.CT_MULTI_VALUE:
```

Comment:
Is this check needed? It looks like `multivalue.parse()` would be have already been called in `value_based_on_content_type()`:
```python
    elif content_type == Protobuf.CT_MULTI_VALUE:
        return multivalue.parse(value)
```